addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
Sorry for being so late to reply! I'd be in favor of keeping `infinityModel` as a parameter because we don't necessarily want users of this addon to have to think deeply about implementation details of `ember-infinity`, instead we just want them to specify how the response from the server (`infinityModel`) should update the route's data (`model`). So from an addon-overrider user experience point of view, I think it's slightly nicer to have it be an argument.

Another argument-argument is that you might change the private variable name later, but you don't necessarily want users of the addon to have to change their code.

But, from a general software point of view, it does make more sense to omit the second argument since it _is_ superfluous. And if you're overriding this method, even though it's public, you maybe know what you're in for.

In any case, I'm happy with either, the main thing is to have a separate method. I'll add something to the README to this effect and whichever you think is better @hhff  we will go with.